docs/instructions.rst

```diff
@@ -216,3 +216,51 @@ For loading data into the database from selected samples only we can proceed as
           As of this moment we do not provide an option for bulk upload of files from a dataset.
 
 .. note:: For loading 1000 genome dataset see: :ref:`genome-dataset` instructions.
+
+.. _optional-features:
+
+Optional Features
+-----------------
+
+The Beacon specification contains some optional features that may be turned on or off.
+
+.. _handover-protocol:
+
+Handover Protocol
+~~~~~~~~~~~~~~~~~
+
+The handover protocol is a feature similar to `HATEOAS and HAL<https://restfulapi.net/hateoas/>`_. It can be used to convey
+extra information regarding the beacon service or the dataset response, that resides in a URL that has been constructed ready for use.
```

Comment:
> constructed ready for use

...this phrase makes me a bit confused. Could it be rephrased to "...constructed for this usage"?

Apart from that, it all looks good to me!

Reply:
I removed the last part of the sentence altogether, and have readers refer to the external sources for more info!